azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -74,6 +74,11 @@ def __del__(self):
         """
         self.amqp_svc_client.disconnect_sync()
 
+    def __ensure_quoted(self, etag):
```

Comment:
Don't do double underscores like this in Python. Many people will say this forces the function or method to be private (as opposed to merely convention-private with a single underscore), but that's incorrect. Python has no concept of private. This use of double underscore instead enables name-mangling, which can appear as making something private, but is actually more complex, and should be avoided unless you really need to be name-mangling.

A single underscore to make this convention-private should suffice.

Also, since this method doesn't use the instance of the object, it probably shouldn't be an instance method. You could make this a classmethod instead, or better yet (IMO), probably just a utility helper function defined outside of the class

Reply:
Done